src/pytest_regressions/ndarrays_regression.py

```diff
@@ -114,7 +120,7 @@ def _check_fn(self, obtained_filename, expected_filename):
         # Compare the contents of the arrays.
         comparison_tables_dict = {}
         for k, obtained_array in obtained_data.items():
-            expected_array = expected_data.get(k)
+            expected_array = expected_data[k]
```

Comment:
Looks like a change in behavior, could it have been untested behavior?

Reply:
No, actually mypy caught this problem: by using `get` here, the type was `Optional[np.ndarray]`, which was then passed on to `_check_data_types` which does not expect an `Optional`. By using `get` here we would be postponing the problem to blow up later, and mypy helped us here.

So no behavior changed, using `[]` we get a type error here rather than a cryptic "None"-problem later.